scripts/get

```diff
@@ -48,25 +58,37 @@ if [ -n "$PKG_URL" -a -n "$PKG_SOURCE_NAME" ]; then
     sleep 1
   done
 
-  if ! [ -f $SOURCES/$1/$PKG_SOURCE_NAME -a "$(cat $STAMP 2>/dev/null)" == "$PKG_URL" ]; then
-    rm -f $SOURCES/$1/$PKG_SOURCE_NAME $STAMP
+  if ! _get_file_already_downloaded $1; then
```

Comment:
No, it's required.

If there are two (or more) processes trying to download the same file at the same time (ie. concurrent builds out of the same repo, which is something I do quite a lot) then only one process will acquire the `flock` in line 56 and then proceed to download the file, while all the other processes will be blocked, spinning slowly in the `while !flock...done` loop.

Once the file is downloaded the process with the `flock` will exit the `get` script and automatically release the `flock`, whereupon one of the other other processes will acquire the `flock` and exit from the `while...done` loop.

This unblocked process will then execute the `if` condition in line 61 and - assuming the file has been downloaded successfully by the first process - avoid doing any download processing at all, dropping out of the script and releasing the `flock`. If the first process did not successfully download the package, then the second process will repeat the same download.